CalibTracker/SiPixelESProducers/plugins/SiPixelQualityESProducer.cc

```diff
@@ -141,4 +142,32 @@ void SiPixelQualityESProducer::setIntervalFor(const edm::eventsetup::EventSetupR
   oValidity = infinity;
 }
 
+void SiPixelQualityESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // siPixelQualityESProducer
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>("siPixelQualityLabel", "");
+  desc.add<std::string>("siPixelQualityLabel_RawToDigi", "");
+  {
+    edm::ParameterSetDescription desc_ps;
+    desc_ps.add<std::string>("record", "SiPixelQualityFromDbRcd");
+    desc_ps.add<std::string>("tag", "");
+    std::vector<edm::ParameterSet> default_ps;
+    default_ps.reserve(2);
+    {
+      edm::ParameterSet temp;
+      temp.addParameter<std::string>("record", "SiPixelQualityFromDbRcd");
+      temp.addParameter<std::string>("tag", "");
+      default_ps.push_back(temp);
+    }
+    {
+      edm::ParameterSet temp;
+      temp.addParameter<std::string>("record", "SiPixelDetVOffRcd");
+      temp.addParameter<std::string>("tag", "");
+      default_ps.push_back(temp);
+    }
+    desc.addVPSet("ListOfRecordToMerge", desc_ps, default_ps);
+  }
+  descriptions.add("siPixelQualityESProducer", desc);
```

Comment:
This could be `addWithDefaultLabel`

Reply:
hi @ferencek given that the whole PR is somewhat "cosmetic" and there is no urgent need for it to get merged, I think it's fine to retrigger tests, it's better to have this done now then to remember it and do it later...